src/Components/Header.jsx

```diff
@@ -14,6 +19,10 @@ export const Header = () => {
           src={process.env.PUBLIC_URL + "/WWCBD.png"}
         ></img>
       </a>
+      <button className="btn btn-main" onClick={() => setShowModal(true)}>
+        Contact Us
```

Comment:
until the submit is set up to send data, could we leave this commented out?

Reply:
Yes. I was originally going to place the button next to the others, but I saw that it was just part of the card container component rather than a nav bar